lib/events/dynamoevents/dynamoevents_test.go

```
@@ -180,14 +184,18 @@ func (s *DynamoeventsSuite) TestEventMigration(c *check.C) {
 	end := start.Add(time.Hour * time.Duration(24*11))
 	attemptWaitFor := time.Minute * 5
 	waitStart := time.Now()
+	var eventArr []event
 
 	for time.Since(waitStart) < attemptWaitFor {
-		events, _, err := s.log.searchEventsRaw(start, end, defaults.Namespace, []string{"test.event"}, 1000, "")
-		sort.Sort(byTimeAndIndexRaw(events))
+		utils.RetryStaticFor(time.Minute*5, time.Second*5, func() error {
```

`RetryStaticFor` error is not checked.

---

Fixed